projects/core/src/lib/components/feature-info/feature-info-template-renderer/feature-info-template-renderer.component.ts

```diff
@@ -0,0 +1,46 @@
+import { ChangeDetectionStrategy, Component, Input, OnChanges, SecurityContext, signal } from '@angular/core';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
+import { FeatureInfoModel } from '../models/feature-info.model';
+import { from, take } from 'rxjs';
+
+@Component({
+  selector: 'tm-feature-info-template-renderer',
+  templateUrl: './feature-info-template-renderer.component.html',
+  styleUrls: ['./feature-info-template-renderer.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  standalone: true,
+})
+export class FeatureInfoTemplateRendererComponent implements OnChanges {
+
+  @Input()
+  public feature: FeatureInfoModel | undefined;
+
+  @Input()
+  public template: string | undefined;
+
+  public rendererTemplate = signal<SafeHtml | null>(null);
+
+  constructor(
+    private sanitizer: DomSanitizer,
+  ) {}
+
+  public ngOnChanges() {
+    from(import('marked'))
+      .pipe(take(1))
+      .subscribe(markedLib => {
+        const replacements: string[] = [];
+        const replacementMap = new Map<string, string>();
+        this.feature?.sortedAttributes.forEach(a => {
+          replacements.push(a.key);
+          replacementMap.set(a.key, a.attributeValue);
+        });
+        const regex = new RegExp('{{\\s*(' + replacements.join('|') + ')\\s*}}', 'gi');
+        const replaced = this.template?.replace(regex, (fullMatch, match) => {
+          return replacementMap.get(match) || "";
```

Comment:
So attribute values can contain markdown and html tags (with javascript code sanitized). Do we want escape some  of this? Or replace after markdown to html conversion?

Reply:
Think it does not harm to allow markdown in attribute values? Should not be a problem from security perspective since we sanitize last?